packages/ui-components/style/icons/data-source.svg

```diff
@@ -0,0 +1 @@
+<svg focusable="false" preserveAspectRatio="xMidYMid meet" xmlns="http://www.w3.org/2000/svg" width="600" height="600" viewBox="0 0 32 32" aria-hidden="true"><polygon points="30 30 24 30 24 28 28 28 28 4 24 4 24 2 30 2 30 30"></polygon><path d="M24.91,19.59a4.51,4.51,0,0,0-2.66-3.24,4.55,4.55,0,0,0-3.5,0,4.94,4.94,0,0,0-.64.35L15.3,13.89a4.53,4.53,0,1,0-1.41,1.41l2.81,2.81a4.48,4.48,0,0,0-.61,3.3,4.51,4.51,0,0,0,2.66,3.24,4.55,4.55,0,0,0,3.5,0,4.51,4.51,0,0,0,2.66-3.24,4.65,4.65,0,0,0,0-1.82ZM11.5,14A2.5,2.5,0,1,1,14,11.5,2.5,2.5,0,0,1,11.5,14Z"></path><polygon points="8 30 2 30 2 2 8 2 8 4 4 4 4 28 8 28 8 30"></polygon><title>Data connected</title></svg>
```

Comment:
I like this icon :)
Not sure if it matters, and it's really a tiny detail that popped up, but I noticed both this one and code snippets icon lines are black, compared to a dark grey in all the other icons listed.
Maybe @ajbozarth has some thoughts if it's ok to leave them as is?
![image](https://user-images.githubusercontent.com/25207344/83079711-67ef9500-a04a-11ea-9871-22a556fd9f61.png)